src/test/scala/beam/agentsim/infrastructure/ChargingNetworkSpec.scala

```diff
@@ -95,7 +95,7 @@ class ChargingNetworkSpec
         val xfcChargingPoint = CustomChargingPoint("ultrafast", 250.0, ElectricCurrentType.DC)
         // first request is handled with the only stall in the system
         val firstInquiry =
-          ParkingInquiry.init(centerSpaceTime, "work", beamVehicle = Some(vehicle1), triggerId = 73737)
+          ParkingInquiry.init(centerSpaceTime, "charge", beamVehicle = Some(vehicle1), triggerId = 73737)
```

Comment:
@haitamlaarabi I changed `activityType` from "work" to "charge" since we return fast charger only if inquiry is not for home/work/overnight. Let me know if this change is valid or not.